src/Elements/Geometry/Polyline.cs

```diff
@@ -159,7 +159,7 @@ private Vector3 PointAtInternal(double u, out int segmentIndex)
                 var a = this._vertices[i];
                 var b = this._vertices[i + 1];
                 var currLength = a.DistanceTo(b);
-                var currVec = (b - a).Normalized();
+                var currVec = (b - a)
```

Comment:
Missing a `;`, which is causing the build to fail. Can you update? Then I'll merge.